<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="911" started-at="2018-11-29T23:53:44Z" finished-at="2018-11-29T23:53:45Z">
    <groups>
      <group name="loginTrue">
        <method signature="LoginTest.loginTrue()[pri:0, instance:wj.test.httpclient.cases.LoginTest@5e3a8624]" name="loginTrue" class="wj.test.httpclient.cases.LoginTest"/>
      </group> <!-- loginTrue -->
      <group name="loginFalse">
        <method signature="LoginTest.loginFalse()[pri:0, instance:wj.test.httpclient.cases.LoginTest@5e3a8624]" name="loginFalse" class="wj.test.httpclient.cases.LoginTest"/>
      </group> <!-- loginFalse -->
    </groups>
    <test name="Default test" duration-ms="911" started-at="2018-11-29T23:53:44Z" finished-at="2018-11-29T23:53:45Z">
      <class name="wj.test.httpclient.cases.LoginTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:wj.test.httpclient.cases.LoginTest@5e3a8624]" name="beforeTest" is-config="true" duration-ms="53" started-at="2018-11-29T23:53:44Z" description="测试准备工作" finished-at="2018-11-29T23:53:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="loginFalse()[pri:0, instance:wj.test.httpclient.cases.LoginTest@5e3a8624]" name="loginFalse" duration-ms="779" started-at="2018-11-29T23:53:44Z" description="用户登录成功" finished-at="2018-11-29T23:53:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at wj.test.httpclient.cases.LoginTest.loginFalse(LoginTest.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginFalse -->
        <test-method status="PASS" signature="loginTrue()[pri:0, instance:wj.test.httpclient.cases.LoginTest@5e3a8624]" name="loginTrue" duration-ms="52" started-at="2018-11-29T23:53:45Z" description="用户登录成功" finished-at="2018-11-29T23:53:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTrue -->
      </class> <!-- wj.test.httpclient.cases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
